scripts/build-production

```diff
@@ -30,4 +30,4 @@ EOF
 rm -rf "${BUILD_OUTPUT}"
 dotnet publish "src/${PROJECT}/${PROJECT}.csproj" -c "${CONFIGURATION}" -r "${DEPLOY_RUNTIME}" \
   -o "${BUILD_OUTPUT}/app" /p:Version="${BUILD_NUMBER}" /p:AngularConfig="${ANGULAR_CONFIG}" \
-  --self-contained
+  --self-contained true
```

Comment:
Doesn't this enable self-contained? `--no-self-contained` disables self-contained.

Reply:
Thank you. Looks like I forgot to _save_ my file again after some testing.